epub33/a11y-tech/index.html

```diff
@@ -133,50 +134,28 @@ <h3>Purpose and Scope</h3>
 			<section id="sec-terminology">
 				<h3>Terminology</h3>
 
-				<p>This document uses the following terminology defined in EPUB 3 [[EPUB-3]]:</p>
+				<p>This document uses terminology defined in EPUB 3 [[EPUB-3]]:</p>
 
-				<ul>
-					<li>
-						<a href="https://www.w3.org/TR/epub/#dfn-epub-content-document">EPUB Content Document</a>
-					</li>
-					<li>
-						<a href="https://www.w3.org/TR/epub/#dfn-epub-creator">EPUB Creator</a>
-					</li>
-					<li>
-						<a href="https://www.w3.org/TR/epub/#dfn-epub-navigation-document">EPUB Navigation Document</a>
-					</li>
-					<li>
-						<a href="https://www.w3.org/TR/epub/#dfn-epub-publication">EPUB Publication</a>
-					</li>
-					<li>
-						<a href="https://www.w3.org/TR/epub/#dfn-media-overlay-document">Media Overlays Document</a>
-					</li>
-					<li>
-						<a href="https://www.w3.org/TR/epub/#dfn-package-document">Package Document</a>
-					</li>
-					<li>
-						<a href="https://www.w3.org/TR/epub/#dfn-epub-reading-system">Reading System</a>
-					</li>
-				</ul>
+				<p>In addition, it uses the <a href="https://www.w3.org/TR/WCAG2/#dfn-assistive-technologies">WCAG 2
```

Comment:
Wouldn't it be cleaner to use data-cite for WCAG2 as well (here and everywhere else)?

Reply:
Does it work for the "latest version" links?

I'll test and see and if it works I'll update.